### PR TITLE
refine export cleaner

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
fix https://github.com/pingcap/tidb-operator/issues/2914

### What is changed and how does it work?
when rclone meet `doesn't exist` error, make cleaner just ignore it.

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
test delete BR and dumpling's backup file and then delete backup CR.

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
Return normally when we clean data but backup data is not found.
```

### CODE REVIEW

File: cmd/backup-manager/app/clean/clean.go [hunk 1]
```diff
@@ -67,7 +68,7 @@ func (bo *Options) cleanRemoteBackupData(bucket string, opts []string) error {
 	destBucket := util.NormalizeBucketURI(bucket)
 	args := util.ConstructArgs(constants.RcloneConfigArg, opts, "deletefile", destBucket, "")
 	output, err := exec.Command("rclone", args...).CombinedOutput()
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "doesn't exist") {
```
Comment (cofyc) [inline cmd/backup-manager/app/clean/clean.go:71]: does `rclone` document its exit codes?

Comment (cofyc) [reply cmd/backup-manager/app/clean/clean.go:71]: checking the error string is fragile, can we use the exit codes [documented here](https://rclone.org/docs/#list-of-exit-codes)?

Comment (lichunzhu) [reply cmd/backup-manager/app/clean/clean.go:71]:
```log
E0710 03:41:09.430348       1 manager.go:79] clean cluster demo/basic-backup backup s3://chunzhu/backup-2020-07-09T06:31:00Z.tgz failed, err: cluster demo/basic-backup, execute rclone deletefile command failed, output: 2020/07/10 03:41:09 ERROR : Attempt 1/3 failed with 1 errors and: s3:chunzhu/backup-2020-07-09T06:31:00Z.tgz is a directory or doesn't exist
```
It sames that `rclone` dosen't document exit codes in command lines.

Comment (lichunzhu) [reply cmd/backup-manager/app/clean/clean.go:71]: > checking the error string is fragile, can we use the exit codes [documented here](https://rclone.org/docs/#list-of-exit-codes)?

nice advise! I will take a try

**Pull request merged**